puffin_egui/src/lib.rs

```diff
@@ -689,7 +689,7 @@ impl ProfilerUi {
             };
 
             let frame_rect = Rect::from_min_max(
-                Pos2::new(x, rect.top()),
+                Pos2::new(x - 0.5 * frame_spacing, rect.top() + 0.5 * frame_spacing),
                 Pos2::new(x + frame_width, rect.bottom()),
             );
```

Comment:
Could _maybe_ move the `expand2` up to here, like:
```suggestion
            )
            .expand2(vec2(0.5 * frame_spacing, 0.0))
```

but inlining the expansion is probably fine too

Reply:
You mean instead of doing the subtraction and addition on line 692?

Reply:
right - my impression was that this inlined the equivalent of doing the `.expand2` which was removed where checking if the `mouse_pos` is contained in the frame.

Reply:
Yeah I can change that, I also noticed the addition is wrong as it adds to the Y component of the min position. So changing to `.expand2` seems like a good solution 👍

Reply:
but on second glance maybe not - this now just expands the `min` position not the `max` position, so `expand2` wouldn't be equivalent.

Reply:
The function not expanding max as well is what's wrong that I mentioned in my comment above, I'm fixing that as well.

Reply:
yeah, I realized afterwards that the bars were getting filled out visually if I tried using expand2 up-front and yeah the follow up to reduce the visual rect makes sense